openedx/core/djangoapps/user_api/accounts/api.py

```diff
@@ -517,3 +521,37 @@ def _validate_email(email):
         raise AccountEmailInvalid(
             u"Email '{email}' format is not valid".format(email=email)
         )
+
+def set_has_profile_image(username, has_profile_image=True):
```

Comment:
It's a shame we can't just use update_account_settings. I suppose making has_profile_image writeable in general is bad.

I think I would just put this helper method in the same file as where it is being used-- specific to file uploads. Having this method here seems almost as bad as just allowing editing through update_account_settings.

And you might as well pass in the user instead of the username-- you always have the requesting user.